packages/benchmark/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "n8n-benchmark",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index",
+  "scripts": {
+    "build": "tsc -p tsconfig.build.json && tsc-alias -p tsconfig.build.json",
+    "start": "./bin/n8n-benchmark",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "typecheck": "tsc --noEmit",
+    "watch": "concurrently \"tsc -w -p tsconfig.build.json\" \"tsc-alias -w -p tsconfig.build.json\""
```

Comment:
I thought we had nodemon for watching?

Reply:
That was leftover from development. Removed it now

Reply:
Can we use the new watch flag that comes with node out of the box?

https://medium.com/@moaidmoaidrazhy/node-watch-vs-nodemon-4c8f0fc8a1af